lms/views/helpers/frontend_app.py

```diff
@@ -0,0 +1,21 @@
+"""Helpers for configuring the front-end JavaScript application."""
+
+from lms.views.helpers._authorization import is_instructor
+
+__all__ = ("configure_grading",)
+
+
+def configure_grading(request, js_config):
+    """
+    Insert any needed JS context to configure the front end for grading.
+
+    This is only enabled for instructors on BlackBoard for now. Note that this
+    is entirely distinct from Canvas Speedgrader, which provides its own UI.
+    """
+    if is_instructor(request) and _is_blackboard(request):
+        js_config["lmsGrader"] = True
+
+
+def _is_blackboard(request):
```

Comment:
I suspect this may morph one day in to `_is_not_canvas`

Reply:
Indeed! Part of my thinking in making it a private method here.

Reply:
I also think this probably belongs somewhere reusable as the same general logic is used by the record-upserting view decorator.